src/components/select-field/SelectDropdownList.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import type { SelectOptionValueProp } from './props';
+
+export const OVERLAY_SCROLLABLE_CLASS = 'bdl-SelectField-overlay--scrollable';
+
+type Props = {
+    children: React.Node,
+    isScrollable?: boolean,
+    multiple?: boolean,
+    placement: string,
```

Comment:
Seems like this could/should be optional with a default value?

Reply:
This `placement` is just a description used to fill the `data-placement` attribute of the `<ul>`, so it seems like if it's optional without a default value that should be fine. Thoughts?